eZ/Publish/Core/Persistence/Cache/ContentTypeHandler.php

```diff
@@ -239,9 +239,9 @@ static function (Type $type) use ($status, $getTypeKeysFn) {
     public function loadByIdentifier($identifier)
     {
         return $this->getCacheValue(
-            $identifier,
+            $this->escapeForCacheKey($identifier),
```

Comment:
Shouldn't `$this->escapeForCacheKey($identifier)` be moved higher to avoid calling it twice?